extension/app/ts/background/backgroundUtils.ts

```diff
@@ -20,7 +20,7 @@ export async function sendPopupMessageToOpenWindows(message: MessageToPopup) {
 			if (error?.message?.includes('A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received')) {
 				return
 			}
-			if (error) return console.error(`Popup message error: ${ error.message }`);
+			if (error) return console.error(`Popup message error: ${error.message}`);
```

Comment:
on this repo we always have spaces inside curly brackets

Reply:
Much to Micah's annoyance.